src/WarcraftLegacies.Source/Quests/Cthun/QuestTanarisOutpost.cs

```diff
@@ -21,7 +21,7 @@ public sealed class QuestTanarisOutpost : QuestData
     /// </summary>
     /// <param name="rescueRect">Units in this area will start invulnerable and be rescued when the quest is complete.</param>
     public QuestTanarisOutpost(Rectangle rescueRect) : base("The Noxious Lair",
-      "The sands of Tanaris call for the Hive, we needto eliminate the Trolls of Zul'farrak first.",
+      "The sands of Tanaris call for the Hive, we need to eliminate the Trolls of Zul'farrak first.",
```

Comment:
```suggestion
      "The sands of Tanaris call for the Hive, but first I must eliminate the Trolls of Zul'farrak.",
```
Just to keep the first-person vibe consistent.